src/command_modules/azure-cli-consumption/azure/cli/command_modules/consumption/_validators.py

```diff
@@ -30,6 +30,15 @@ def validate_both_start_end_dates(namespace):
         raise CLIError("usage error: Both --start-date and --end-date need to be supplied or neither.")
 
 
+def validate_usage_bp_inputs(namespace):
+    """Validates the existence of both start and end dates in the parameter or neither"""
+    if (namespace.start_date and not namespace.end_date) or (namespace.end_date and not namespace.start_date):
```

Comment:
Recommend the more concise: 
`if bool(namespace.start_date) != bool(namespace.end_date):`